client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -74,21 +75,31 @@ const CancelPurchaseRefundInformation = ( { purchase, includedDomainPurchase } )
 					),
 					i18n.translate(
 						'You will receive a partial refund of %(refundAmount)s which is %(planCost)s for the plan ' +
-						'minus %(domainCost)s for the domain. To cancel the domain with the plan and ask for a full ' +
-						'refund, please {{contactLink}}contact support{{/contactLink}}.',
+						'minus %(domainCost)s for the domain.',
 						{
 							args: {
 								domainCost: includedDomainPurchase.priceText,
 								planCost: purchase.priceText,
 								refundAmount: purchase.refundText
-							},
-							components: {
-								contactLink: <a href={ support.CALYPSO_CONTACT } />
 							}
 						}
 					)
 				];
 
+				if ( moment().isBefore( moment( includedDomainPurchase.subscribedDate ).add( refundPeriodInDays, 'days' ) ) ) {
```

Comment:
@aidvu Instead of replicating business logic like this on the client, can we use the `is_refundable` property of the domain purchase?